integration/skaffold/helper.go

```diff
@@ -169,7 +169,7 @@ func (b *RunBuilder) WithProfiles(profiles []string) *RunBuilder {
 
 // RunBackground runs the skaffold command in the background.  The Skaffold output
 // is accumulated and logged on test failure.
-func (b *RunBuilder) RunBackground(t *testing.T) io.Reader {
+func (b *RunBuilder) RunBackground(t *testing.T) {
 	t.Helper()
 	out := bytes.Buffer{}
```

Comment:
can be replaced with `ioutil.Discard`

Reply:
No, we need it to log output on failure.  It's painful to diagnose failed tests otherwise.

Reply:
you're right. I didn't look beyond this function. My bad